cov.py

```diff
@@ -268,6 +268,8 @@ def main():
   else:
     args.testlist = args.custom_target + "/testlist.yaml"
 
+  # args.testlist = cwd + "/yaml/cov_testlist.yaml" ## needed if need to force
```

Comment:
I still feel this one is needed. Do you want to a custom testlist for the coverage? I think we may need to remove line 268-269 as well. Otherwise the user may pick up the wrong testlist.

Reply:
as I said before - we cannot run our regressions here with the logic of the target, custom_target, settings_dir, isa, etc... options... and to run I uncomment this line and force things - when you sort the configuration for DUT this line can be removed.

Reply:
The intention of this line is to force the teslist of the coverage collection test to be ...../riscv-dv/yaml/cov_testlist.yaml, regardless of what target you are using. Is this something not expected to you? The testlist under target directory is used for normal regression, not for functional coverage collection

Reply:
My point is that this line should be uncommented by default, like what you did in your run.

Reply:
ok, sorry, if it is un-commented - then it forces things - if that is ok for you it is good for me -- how do I proceed - will you just un-comment?

Reply:
you can uncomment it, then
git add .
git commit --amend
git push origin HEAD -f

Reply:
OK I did this - now what happens? thanks

Reply:
thanks, Approved the PR, I will merge it to mainline.